docs/examples/auto-instrumentation/README.rst

```diff
@@ -0,0 +1,142 @@
+Autoinstrumentation
+===================
+
+Overview
+--------
+
+This example shows how to use auto-instrumentation in OpenTelemetry.
+This example is also based on a previous example for OpenTracing that
+can be found `here <https://github.com/yurishkuro/opentracing-tutorial/tree/master/python>`_.
+
+The source files of these examples are available :scm_web:`here <docs/examples/auto-instrumentation/>`.
+
+This example uses 2 scripts whose main difference is they being
+instrumented manually or not
+
+1. ``server_instrumented.py`` which has been instrumented manually
+2. ``server_uninstrumented.py`` which has not been instrumented manually
+
+The former will be run without the automatic instrumentation agent and
+the latter with the automatic instrumentation agent. They should produce
+the same result, showing that the automatic instrumentation agent does
+the equivalent of what manual instrumentation does.
+
+In order to understand this better, here is the relevant part of both
+scripts:
+
+Manually instrumented server
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: python
+
+    # server_instrumented.py
+    @app.route("/server_request")
+    def server_request():
+        with tracer.start_as_current_span(
+            "server_request",
+            parent=propagators.extract(
+                lambda dict_, key: dict_.get(key, []), request.headers
+            )["current-span"],
+        ):
+            print(request.args.get("param"))
+            return "served"
+
+Publisher not instrumented manually
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: python
+
+    # server_uninstrumented.py
+    @app.route("/server_request")
+    def server_request():
+        print(request.args.get("param"))
+        return "served"
+
+Preparation
+-----------
+
+This example will be executed in a separate virtual environment:
+
+.. code-block:: sh
+
+    $ mkdir auto_instrumentation
+    $ virtualenv auto_instrumentation
+    $ source auto_instrumentation/bin/activate
+
+Installation
+------------
+
+.. code-block:: sh
+
+    $ pip install opentelemetry-api
+    $ pip install opentelemetry-sdk
+    $ pip install opentelemetry-auto-instrumentation
+    $ pip install ext/opentelemetry-ext-flask
```

Comment:
should this just be `pip install opentelemetry-ext-flask`? as there is no step above saying that the repo needs to be cloned, or you need to be in the root.